.gitignore

```
@@ -1,2 +1,3 @@
 vendor
 **/*.hhast.parser-cache
+composer.lock
```

```suggestion
composer.lock

```
Though, don't we want to track `composer.lock`?

---

I am afraid you can't do that.
Take a look at the other Hack libraries
They all git ignore the lock file.
Travis should not be able to see it, otherwise the tests fail.
Composer install would install the old dependencies in newer versions of hhvm.